datahub-web-react/src/app/entity/user/UserInfoSideBar.tsx

```diff
@@ -91,14 +101,30 @@ export default function UserInfoSideBar({ sideBarData, refetch }: Props) {
                 refetch();
             });
     };
+    const dataHubRoleName = dataHubRoles && dataHubRoles.length > 0 && (dataHubRoles[0]?.entity as DataHubRole).name;
+
+    const mapRoleIcon = (roleName) => {
```

Comment:
This can be extracted to somewhere outside of the component (utils or above the component function)

Reply:
Ideally some roleUtils file that is clear to extend

Reply:
Since we'll likely need to reuse this in other places, right?